src/controllers/product.controller.ts

```
@@ -581,6 +585,204 @@ export const deleteProductImage = async (req: Request, res: Response): Promise<v
     }
 };
 
+// Create product variant
+export const createProductVariant = async (
+    req: Request<{ id: string }, {}, IProductVariantCreate>,
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)